config/settings/base.py

```diff
@@ -609,6 +609,42 @@
     },
 }
 
+
+def json_keys_to_float(json_dict):
+    if isinstance(json_dict, dict):
+        processed_json_dict = {}
+        for key, value in json_dict.items():
+            try:
+                processed_key = float(key)
+            except ValueError:
+                processed_key = None
+            processed_json_dict[processed_key] = value
+        return processed_json_dict
+    return json_dict
+
+
+def preprocess_sprints_hours_reserved_for_review(json_dict):
```

Comment:
btw, created this method were I "enforce" the null value being defined and also casts the strings keys to float values(and None)

Reply:
```suggestion
def validate_sprints_hours_reserved_for_review(json_dict):
```
What do you think?